context/context.go

```diff
@@ -43,6 +46,21 @@ func initOpVersion() {
 	OpVersion = MaxOpVersion
 }
 
+func initEtcdClient() error {
+	Hostname, err := os.Hostname()
```

Comment:
It should not be done in doInit because i am using context.Hostname before doInit() call.